webpack.config.js

```diff
@@ -25,4 +25,22 @@ module.exports = {
 		profile: path.resolve(process.cwd(), 'src/profile', 'index.js'),
 		profile_style: path.resolve(process.cwd(), 'src/profile', 'style.scss'),
 	},
+	module: {
```

Comment:
I might change this rule to be explicit for `marker-icon-2x.png` and `marker-shadow.png`. These are the only 2 images that cannot get a hash or Leaflet will not work. I'll play with it a bit, but this is fine for right now.

Reply:
Ok, this is done.